examples/notebooks/14Lossy_splices.ipynb

```diff
@@ -246,7 +246,7 @@
     "    ast_var=ast_var,\n",
     "    rst_var=rst_var,\n",
     "    rast_var=rast_var,\n",
-    "    transient_asym_att_x=[50.],\n",
+    "    trans_att=[50.],\n",
```

Comment:
`trans_att` is now a property of `DataStore`, right? Perhaps it's better to use it that way in the example notebooks as well, instead of passing it as a kwarg.

Reply:
Well, this is kind of the point. It has to be either a property or an xarray coordinate object. I chose the latter, because the result comprises of two time series (forward and backward) per trans_att location. Which makes it convenient to plot the results (similar to storing `ds.df`, which also has a time dimension).
And since it is not a property, we can not set it as such. That is why I created a helper function to set the trans_att coordinates, and delete all dataarrays that made use of the old trans_att. ( `ds.set_trans_att()`) This helper function gets called at the beginning of the calibration routine.

Reply:
Ah yes, I get it now!